Public/Src/App/Bxl/Tracing/Log.cs

```diff
@@ -54,7 +54,7 @@ public abstract partial class Logger
             // Prevent this from going to the log. It is only for ETW and telemetry. DominoInvocationForLocalLog is for the log.
             Keywords = (int)Keywords.SelectivelyEnabled,
             Message = AppInvocationMessage)]
-        public abstract void DominoInvocation(LoggingContext context, string commandLine, BuildInfo buildInfo, MachineInfo machineInfo, string sessionIdentifier, string relatedSessionIdentifier, string startupDirectory, string mainConfig);
+        public abstract void DominoInvocation(LoggingContext context, string commandLine, BuildInfo buildInfo, MachineInfo machineInfo, string sessionIdentifier, string relatedSessionIdentifier, string startupDirectory, string mainConfig, bool isDotNetCore);

```

Comment:
The comment for this event says
> WDG has Asimov telemetry listening to this event. Any change to an existing field will require a breaking change announcement

Adding a new field should be backward compatible and thus not constitute a breaking change.

Reply:
👍